examples/baseball.py

```diff
@@ -373,6 +373,9 @@ def main(args):
                         help="run this example in GPU")
     args = parser.parse_args()
 
+    # work around the error "CUDA error: initialization error" when arg.cuda is False
+    # see https://github.com/pytorch/pytorch/issues/2517
+    torch.multiprocessing.set_start_method("spawn")
```

Comment:
When using multiprocessing in CPU, if pytorch is installed with cuda support, we should "avoid any call to cuda prior to starting subprocesses". In this case, one of the call is `torch.manual_seed(...)` statement.